sdk/include/aditof/sensor_definitions.h

```diff
@@ -45,12 +45,23 @@ namespace aditof {
  * @struct SensorDetails
  * @brief Provides details about the device
  */
-struct SensorDetails {
+struct SensorInformations {
```

Comment:
This seems to be used internally only. For this reason it shouldn't be public. Also you could have directly 2 members (id and name) instead of this struct.